libs/SalesforceSDK/src/com/salesforce/androidsdk/ui/PasscodeActivity.java

```diff
@@ -188,6 +197,15 @@ public void onClick(View v) {
                 passcodeField.setText(inputText.trim());
             }
         }
+
+        // Determine biometric hardware, default is fingerprint.
+        if (VERSION.SDK_INT >= VERSION_CODES.Q) {
+            if (getPackageManager().hasSystemFeature(PackageManager.FEATURE_FACE)) {
```

Comment:
Can Face and Iris unlock be available on the same device?

Reply:
I don't believe so.  We can decide if we want to update the logic to allow choice or something in the future _if_ a device is ever release that has more than 1 of the 3.